Nightly failure: sanity/block_sync.py

Fails consistently. Example runs: 

http://nightly.neartest.com/run/2cef4105736ab969470df2751e78fd3f0fc0a81d_200428_102644
http://nightly.neartest.com/run/f625606891dec008a572ca61ec37da500c2a95cd_200428_135923

Failure:

```
Traceback (most recent call last):
  File "tests/sanity/block_sync.py", line 48, in <module>
    assert time.time() - start_time < TIMEOUT, "Block sync timed out"
AssertionError: Block sync timed out
```

--- a/pytest/tests/sanity/block_sync.py
+++ b/pytest/tests/sanity/block_sync.py
@@ -18,7 +18,8 @@ consensus_config1 = {"consensus": {"min_block_production_delay": {"secs": 100, "
 nodes = start_cluster(
     2, 0, 4, None,
     [
-        ["epoch_length", 100], ["validators", 0, "amount", "110000000000000000000000000000000"], ["records", 0, "Account", "account", "locked", "110000000000000000000000000000000"],
+        ["epoch_length", 100], ["num_block_producer_seats", 100], ["num_block_producer_seats_per_shard", [25, 25, 25, 25]],
+        ["validators", 0, "amount", "110000000000000000000000000000000"], ["records", 0, "Account", "account", "locked", "110000000000000000000000000000000"],
         ["total_supply", "3060000000000000000000000000000000"]
     ],
     {0: consensus_config0, 1: consensus_config1}

